cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -761,6 +761,20 @@ static PyObject *get_precision(PyObject *self, PyObject *args)
     return PyLong_FromLong(precision);
 }
 
+static PyObject *get_simulator_product(PyObject *m, PyObject *args)
+{
+    COCOTB_UNUSED(m);
+    COCOTB_UNUSED(args);
+    return Py_BuildValue("s", gpi_get_simulator_product());
```

Comment:
Any reason you're not using the `PyUnicode_FromString` I suggested before? This indirection makes it harder to work out whether this is unicode or bytes IMO.

```suggestion
    return PyUnicode_FromString(gpi_get_simulator_product());
```

It also makes it more consistent with the rest of the file.